Code/Framework/AzToolsFramework/AzToolsFramework/UI/DocumentPropertyEditor/FilterAdapter.h

```diff
@@ -21,10 +21,13 @@ namespace AZ::DocumentPropertyEditor
         ~RowFilterAdapter();
 
         void SetSourceAdapter(DocumentAdapterPtr sourceAdapter);
+        void SetIncludeAllMatchDescendants(bool includeAll);
 
     protected:
         struct MatchInfoNode
         {
+            MatchInfoNode(const MatchInfoNode& nodeToCopy) = delete;
```

Comment:
This seems unnecessary given that MatchInfoNode is already a protected member of the RowFilterAdapter, so it is not as if outside code would attempt to copy it.

Also if copy constructor is to be deleted, then the copy assignment operator should also be as well